staging/prometheus-operator/values.yaml

```diff
@@ -64,6 +64,9 @@ mesosphereResources:
     scheduler: true
     traefik: true
     velero: true
+  ## Additional permissions to grant to a container running in the Prometheus pod.
+  ## See: https://github.com/mesosphere/kubeaddons-extrasteps/tree/master/pkg/prometheus/prometheus.go
+  enableAdditionalPrometheusRBACRules: false
```

Comment:
I defaulted this to false since it should only be run when the extra container is specified/defined in kubeaddons-configs - will flip it to `true` in kubeaddons-configs, lmk if this makes sense